### PR TITLE
organize project metadata files

### PR DESCRIPTION
**Pull request by b97pla**
**What problems does this PR solve?**
Metadata files should be organized and delivered together with the runfolder delivery for projects. This PR include those files in this process.

**How has the changes been tested?**
Unit and integration tests in the test suite. Should be verified in staging before production.

**Reasons for careful code review**
If any of the boxes below are checked, extra careful code review should be inititated.

  - [ ] This PR contains code that could remove data

### CODE REVIEW

File: tests/resources/readme/README.md
Comment (Aratz) [inline tests/resources/readme/README.md:1]: This file is binary, is this on purpose?

**Review by Aratz — APPROVED** [review]
Looks good! I had a question about the binary file but otherwise I think it's good to go :)

File: delivery/repositories/project_repository.py [hunk 1]
```diff
@@ -140,18 +140,19 @@ def project_from_dir(d):
...
 
+            # gather report files for the project from the runfolder
             try:
                 project_files.extend(
                     self.get_report_files(
                         project_path,
                         project_name,
-                        runfolder,
-                        checksums=runfolder.checksums
+                        runfolder
```
[the remark this answers is not in the document]
Comment (b97pla) [reply delivery/repositories/project_repository.py:149]: nice, I did not know that!

File: tests/resources/readme/README.md
Comment (b97pla) [reply tests/resources/readme/README.md:1]: err.. no, I'll have a look

File: delivery/repositories/project_repository.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (b97pla) [reply delivery/repositories/project_repository.py:149]: I'm noting this for the future but it's not feasible to change all occurrences in the code now 😅

**Comment by b97pla** [comment]
thanks @Aratz!

**Pull request merged**